### PR TITLE
Awesome Books refactored to use modules

### PR DESCRIPTION
**Pull request by NoelLincoln**
Hello 👋

I have refactored my code to use es6 modules and arrow functions.
I have also used luxon dependency to get the current date and time.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import BookManager from './modules/BookManager.js';
+import GetTime from './modules/GetTime.js';
+import navHandler from './modules/NavigationHandler.js';
+
+function initializeApp() {
```
Comment (fobadara) [inline index.js]: [REQUIRED] Kindly use arrow function. It is a requirement for the milestone.

File: modules/GetTime.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { DateTime } from './Luxon.js';
+
+export default function GetTime() {
```
Comment (fobadara) [inline modules/GetTime.js]: [REQUIRED] Kindly use arrow function. It is a requirement for the milestone.

File: modules/NavigationHandler.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+export default function navHandler() {
```
Comment (fobadara) [inline modules/NavigationHandler.js]: [REQUIRED] Kindly use arrow function. It is a requirement for the milestone.

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @NoelLincoln , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.

REDO

## Highlights
✅ Descriptive README
✅ Descriptive PR summary
✅ Clean code
✅ Good first attempt
✅ Good implementation of the principle of separating concerns.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will improve your application._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KanzaTahreem — APPROVED** [review]
# Approved 🙌

Hi @NoelLincoln 

Good job for implemented the changes suggested by previous reviewerpt 💪 **_Your project is COMPLETE! There is nothing else to say other than... it's time to merge it! 🚀 Congratulations!_**



## Highlights 🎯
There are some excellent points that you have made perfectly. Keep up the great work! 👏

-  ✅ `arrow functions` are used
-  ✅ Used correct GitHub Flow 
-  ✅ Linters are working properly

## Optional suggestions ❕
N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**